manifests/0700_cr.yaml

```diff
@@ -1,8 +1,100 @@
-apiVersion: nfd.kubernetes.io/v1alpha1
+apiVersion: nfd.kubernetes.io/v1
 kind: NodeFeatureDiscovery
 metadata:
   name: nfd-master-server
   namespace: REPLACE_NAMESPACE
 spec:
-  namespace: node-feature-discovery-operator
-  image: k8s.gcr.io/nfd/node-feature-discovery:v0.6.0
+  operand:
+    namespace: node-feature-discovery-operator
+    image: node-feature-discovery
+    repository: k8s.gcr.io/nfd
+    version: v0.6.0
+    imagePullPolicy: Always
+  workerConfig:
+    config: |    
```

Comment:
Should this be `configData` to reflect the latest changes(?)

Reply:
epic face palm, it does